components/NavBar/index.tsx

```diff
@@ -2,37 +2,27 @@ import { useCallback, useState } from 'react'
 import Link from 'next/link'
 import dynamic from 'next/dynamic'
 import Image from 'next/image'
-import NavLink, { NavLinkProps } from './NavLink'
+import DropDownMenu from './DropDownMenu'
 import logo from '../../public/images/foodbank-logo.webp'
+import { NavLinkProps } from './NavLink'
 
 const Auth = dynamic(() => import('../Auth'), { ssr: false })
 
-const links: Array<NavLinkProps> = [
-  {
-    page: 'Super Hero Foods',
-    route: '/'
-  },
-  {
-    page: 'Recipe',
-    route: '/'
-  },
-  {
-    page: 'Progress',
-    route: '/'
-  }
-]
+interface NavbarProps {
+  links: Array<NavLinkProps>
+}
 
-const Navbar = () => {
+const Navbar = ({ links }: NavbarProps) => {
   const [signIn, setSignIn] = useState(false)
   const toggleSignIn = useCallback(() => {
     setSignIn(prev => !prev)
   }, [])
 
   return (
-    <header className='fixed inset-x-0 top-0 z-10 hidden py-3 bg-primary lg:block'>
+    <header className='fixed inset-x-0 top-0 z-10 hidden py-3 bg-primary md:block'>
       <div className='container flex justify-between px-3 mx-auto'>
         <Link href='/'>
-          <a className='relative w-12 h-10 hover:opacity-75'>
+          <a className='absolute left-10 top-2 w-20 h-14 hover:opacity-75'>
```

Comment:
u have modified the dimensions of the image but haven't changed the class that positions the rest of the page content. Also not sure why the navbar needs to be so big